packages/disploy/src/structs/BaseInteraction.ts

```diff
@@ -45,9 +45,12 @@ export class BaseInteraction extends Base {
 	 * The guild of the interaction.
 	 */
 	public guild!: ToBeFetched<Guild> | null;
-
+	/**
+	 * The raw data of the interaction.
+	 */
+	 declare public raw: APIInteraction;
 	public constructor(app: App, raw: APIInteraction) {
-		super(app);
+		super(app, raw);
```

Comment:
While this would probably be a good idea to let the user access the raw objects; it is out of scope for this PR.